lib/dataSources.js

```diff
@@ -167,6 +167,18 @@ const refreshMetadata = (conn, name, tableName = '') => {
   ).then(httpBody);
 };
 
+const suggestions = (conn, trigBody) => {
+  const headers = conn.headers();
+  headers.set('Content-Type', 'application/trig');
+  headers.set('Accept', 'application/ld+json');
```

Comment:
i would maybe make this configurable (`(conn, config, content)`)

Reply:
also, can you update the docs with this?

Reply:
configurable how? into fetch? why?

Reply:
in case we want a return type that is not `ld+json`

Reply:
[`d2c4e12` (#276)](https://github.com/stardog-union/stardog.js/pull/276/commits/d2c4e12ba7bd3b9871d1076ce2ff4f6fd475cac7)